lib/dry/inflector.rb

```diff
@@ -268,6 +268,18 @@ def uncountable?(input)
       !(input =~ /\A[[:space:]]*\z/).nil? || inflections.uncountables.include?(input.downcase)
     end
 
+    def check_for_acronyms(word)
```

Comment:
Seems strange that this method name starts with `check_` but doesn't return a `boolean` value, maybe `acronym_or_capitalize` would be a better name?